.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+## Proposed changes
+
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.\
+If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+## Types of changes
+
+What types of changes does your code introduce to the php Github AApi?
+_Put an `x` in the boxe(s) that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] Improvement (non-breaking change which improves an existing code/behavior)
+- [ ] Deprecated third party dependency update
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation/Typo/Resource/Annotation update that does not involve any direct code modification
+- [ ] Other, please specify: 
+
+## Agreement
+
+I have read the [CONTRIBUTING](https://github.com/KnpLabs/php-github-api/blob/master/CONTRIBUTING.md) and [CODING GUIDELINE](https://github.com/KnpLabs/php-github-api/blob/master/CODING_GUIDELINE.md) docs
+
+## Further comments
+
+If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...
```

Comment:
Hm. Im hesitant.

Reply:
What's bothering you?

Reply:
If you look at the 50 last PRs to this repo. How many of them would put something here?

Reply:
This one 🤓 
Seriously, here again I can remove it if you think that this does not add any value at all.

Reply:
If something only helps 2% of the PRs I do not think it brings enough value. I would remove it.